matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -724,6 +724,18 @@ internal class LocalEchoEventFactory @Inject constructor(
         }
     }
 
+    private fun bodyForReply(timelineEvent: TimelineEvent, isRedactedEvent: Boolean = false): TextContent {
+        val content = when (timelineEvent.root.getClearType()) {
+            in EventType.POLL_END.values -> {
+                localEchoRepository
```

Comment:
I think we should have a fallback for when the related poll event is not found in the repository.
```suggestion
                localEchoRepository
                        .getRelatedPollEvent(timelineEvent)
                        ?.getLastMessageContent()
                        ?: timelineEvent.getLastMessageContent()
```

Reply:
I agree that last message may also be null.

Reply:
Added fallback in [8fb38f5](https://github.com/vector-im/element-android/pull/7969/commits/8fb38f57dcab7f30f9c781bde42165bbb0dff797)